genotype/load/vcf.py

```diff
@@ -64,4 +64,13 @@ def fetch_snp(vcf: VCF, snp: SNP) -> Variant:
         variant = variants[0]
         return variant
     # weird SNP position lookup, not even possible; right?
-    raise ValueError("multiple variants found for SNP")
+    raise ValueError(
+        f"Multiple variants ({len(variants)}) found for SNP at position {pos_str}: {return_alternatives(variants)}."
```

Comment:
Use named args whenever possible. It makes it more clear on what to supply to the function